content/docs/v1.10/_index.md

```diff
@@ -3,9 +3,9 @@ title: "Overview"
 weight: -1
 toc_include: false
 aliases:
-    - /docs/master/index.html
+    - /docs/v1.10/index.html
 cascade:
-    version: master
+    version: "1.10"
```

Comment:
@plumbis i believe all of these fixes will need to happen in the Crossplane repo or else they will get overwritten when syncing -- but perhaps I'm misinterpreting